packages/cli/src/commands/setup/ui/libraries/windicss.js

```diff
@@ -0,0 +1,146 @@
+
+  const tasks = new Listr([
+    {
+      title: 'Installing packages...',
+      skip: () => !install,
+      task: () => {
+        return new Listr([
+          {
+            title: `Install ${packages.join(', ')}`,
+            task: async () => {
+              await execa('yarn', [
+                'workspace',
+                'web',
+                'add',
+                '-D',
+                ...packages,
+              ])
+            },
+          },
+        ])
+      },
+    },
+    {
+      title: 'Setup Webpack...',
+      task: () => {
+        return new Listr([
+          {
+            title: 'Setup Webpack',
+            task: async () => {
+              await execa('yarn', ['redwood', 'setup', 'webpack'])
+            },
+          },
+          {
+            title: 'Configure WindiCSS',
+            task: async () => {
+              const webpackConfigPath = path.join(
+                rwPaths.web.config,
+                'webpack.config.js'
+              )
+              const webpackConfig = fs.readFileSync(webpackConfigPath, 'utf-8')
+              const newWebpackConfig =
+                `const WindiCSSWebpackPlugin = require('windicss-webpack-plugin')\n` +
+                webpackConfig.replace(
+                  '// config.plugins.push(YOUR_PLUGIN)',
+                  '// config.plugins.push(YOUR_PLUGIN)\n  config.plugins.push(new WindiCSSWebpackPlugin())'
+                )
+              fs.writeFileSync(webpackConfigPath, newWebpackConfig)
+            },
+          },
+        ])
+      },
+    },
+    {
+      title: 'Initializing WindiCSS...',
+      task: async () => {
+        const windiConfigPath = path.join(rwPaths.web.config, 'windi.config.js')
+
+        if (fs.existsSync(windiConfigPath)) {
+          if (force) {
+            fs.unlinkSync(windiConfigPath)
+          } else {
+            throw new Error(
+              'Windicss config already exists.\nUse --force to override existing config.'
+            )
+          }
+        }
+
+        const windiConfig = `
+          import { defineConfig } from 'windicss/helpers'
+
+          export default defineConfig({
+            extract: {
+              include: ['**/*.{jsx,tsx,css}'],
+              exclude: ['node_modules', '.git'],
+            },
+          })
+        `
+          .split('\n')
+          .map((line) => line.trim())
+          .join('\n')
+        fs.writeFileSync(windiConfigPath, windiConfig)
+      },
+    },
+    {
+      title: `Adding import to ${rwPaths.web.app}...`,
+      task: (_ctx, task) => {
+        const APP_FILE_PATH = rwPaths.web.app
+        const appFile = fs.readFileSync(APP_FILE_PATH, 'utf-8')
+
+        if (windiImportsExist(appFile)) {
+          task.skip('Imports already exist in ' + APP_FILE_PATH)
+        } else {
+          const newAppFile = appFile.replace(
+            /^import 'index\.css'$/m,
+            "import 'index.css'\nimport 'windi.css'"
+          )
```

Comment:
This is strategy used by the [generate dbAuth command](https://github.com/redwoodjs/redwood/blob/051dde34a0e1a68a4920a639bbebf9b03d87967e/packages/cli/src/lib/index.js#L371-L386). The Routes import is pretty stable, and as good as any to target:
```suggestion
          const newAppFile = appFile.replace(
            "import Routes from 'src/Routes'\n",
            "import Routes from 'src/Routes'\n\nimport 'windi.css'"
          )
```